noggin/representation/user.py

```diff
@@ -36,4 +37,4 @@ class User(Representation):
 
     @property
     def name(self):
-        return self.displayname or self.gecos or self.commonname
+        return self.displayname or self.gecos or self.common_name
```

Comment:
Why this change? Is it to conform with how it's used elsewhere, e.g. in FreeIPA?

Reply:
Hmm not really, this change isn't really necessary. I don't know what's best. I'll probably revert it since after all we already have `displayname` without the underscore.